jest.config.js

```diff
@@ -0,0 +1,7 @@
+/** @type {import('ts-jest/dist/types').InitialOptionsTsJest} */
```

Comment:
let's remove this file! since jest config happens in package.json this will just create confusion.

Reply:
done 👍